components/shared/embedder/lib.rs

```diff
@@ -8,9 +8,11 @@ use std::fmt::{Debug, Error, Formatter};
 
 use base::id::{PipelineId, TopLevelBrowsingContextId, WebViewId};
 use crossbeam_channel::{Receiver, Sender};
+use hyper::{HeaderMap,Method,StatusCode};
```

Comment:
These are just re-exports from the http crate. Let's depend on that crate instead.

Reply:
Ok. Changed to http.